test.bat

```diff
@@ -1,7 +1,8 @@
 cd %PROJECTS_HOME%\azure-kusto-python
 call workon kusto
 call pip uninstall azure-kusto-data azure-kusto-ingest -y 
-call pip install ./azure-kusto-data[pandas] ./azure-kusto-ingest[pandas] 
+call pip install -e ./azure-kusto-data[pandas]
+call pip install -e ./azure-kusto-ingest[pandas]
```

Comment:
Is there a reason to separate them?

Reply:
Yes, for some reason the -e flag does not work well with multiple resources